build.gradle

```diff
@@ -594,6 +594,13 @@ project(':iceberg-dell') {
     implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
     compileOnly 'com.emc.ecs:object-client-bundle'
 
+    compileOnly("org.apache.hadoop:hadoop-common") {
```

Comment:
Why do we need the hadoop-common in the compile path for `iceberg-dell` FileIO & Catalog ?  In my mind, we had a several PR to remove all the hadoop dependencies for 3rd party `FileIO` and `Catalog` implementation.

Reply:
The Hadoop dependencies are removed.